### PR TITLE
add pics

### PR DESCRIPTION
**Pull request by shivambitm**
can we add some pics here in readme file ? @ayushii18 
https://drive.google.com/drive/folders/1YdXkK2HUgGWVRfB6BKOA1bdhDxfwsRc3?usp=sharing
Have a look into this..

### CODE REVIEW

**Comment by shivambitm** [comment]
> Good

thank you

**Review by shivambitm — COMMENTED** [review]
@ayushii18 please see this

**Comment by ayushii18** [comment]
> can we add some pics here in readme file ? @ayushii18
> https://drive.google.com/drive/folders/1YdXkK2HUgGWVRfB6BKOA1bdhDxfwsRc3?usp=sharing
> Have a look into this..

> Yes shivam sure

File: virtual.py [hunk 1]
```diff
@@ -16,3 +16,17 @@
...
+        speak("Hello,Good Morning")
+        print("Hello,Good Morning")
+    elif hour>=12 and hour<18:
+        speak("Hello,Good Afternoon")
+        print("Hello,Good Afternoon")
+    else:
+        speak("Hello,Good Evening")
+        print("Hello,Good Evening")
+    
+
```
Comment (ayushii18) [inline virtual.py:32]: add codes for voice recognition also @shivamkumar0611 @sauravmandal255 @santosh-sudo 
All other members please respond on this ..

**Review by ayushii18 — COMMENTED** [review]
add codes for voice recognition also @shivamkumar0611 @sauravmandal255 @santosh-sudo
All other members please respond on this ..

**Pull request merged**